setuptools/tests/integration/test_pip_install_sdist.py

```diff
@@ -27,7 +28,12 @@
 
 pytestmark = pytest.mark.integration
 
-(LATEST,) = Enum("v", "LATEST")
+
+class v(Enum):
+    LATEST = 1
+
+
+(LATEST,) = v
```

Comment:
I'm not well-versed enough in Enums to understand what's going on here, but I'm starting not to like the smell. This new syntax adds a new name to the namespace and 5 lines to the code (3 of which are blank). Avoid changing the semantics and let's just disable mypy where it's too picky about the types.

Reply:
The exact error is: `"Enum" object is not iterable`

And here's why there has been no effort in supporting this form in mypy: https://github.com/python/mypy/issues/8009#issuecomment-558335186
Might be worth opening an issue for this specific feature anyway

pyright does seem to understand it just right.